src/scripts/main.js

```diff
@@ -1,7 +1,37 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  // створюю контейнер повідомлення
+  const notification = document.createElement('div');
+  // додаю клас notification і типу (success, error, warning)
+
+  notification.classList.add('notification', type);
+
+  // встановлення координат
+  notification.style.position = 'absolute';
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  // додаю заголовок
+  const titleElement = document.createElement('h3');
```

Comment:
The task specifies that the title should preferably be an `h2` element. Please change the `h3` element to `h2` to meet the task requirements.